src/wrapped_radix_engine_toolkit/low_level/compiled_notarized_intent.rs

```diff
@@ -1,3 +1,116 @@
 use crate::prelude::*;
 
-dummy_sargon!(CompiledNotarizedIntent);
+use radix_engine_toolkit::functions::notarized_transaction::{
+    compile as RET_compile_notarized_tx, decompile as RET_decompile_notarize_tx,
+};
+
+use transaction::model::{
+    NotarizedTransactionV1 as ScryptoNotarizedTransaction,
+    SignedIntentV1 as ScryptoSignedIntent,
+};
+
+#[derive(
+    Serialize,
+    Deserialize,
+    Clone,
+    Debug,
+    PartialEq,
+    Eq,
+    Hash,
+    derive_more::Display,
+    derive_more::FromStr,
+    uniffi::Record,
+)]
+pub struct CompiledNotarizedIntent {
```

Comment:
new "strong type" instead of `BagOfBytes` acting as return type for `NotarizedTransaction::compile` (also used for `decompile`)